test/test.py

```diff
@@ -129,7 +132,11 @@ def testClient(self):
                 self.assertEqual(len(subset), docs.total)
                 ids = [x.id for x in docs.docs]
                 self.assertEqual(set(ids), set(subset))
-                
+
+                for doc in client.search(Query('henry king').returning('play', 'nonexist')).docs:
+                    doc.nonexist
```

Comment:
you do know that doc.nonexist should be None and that doc.play should have content in it